shared/notifications/index.js

```diff
@@ -346,5 +367,31 @@ export const getTranslatedUINotifications = (t, locale) => {
           )
         : '',
     },
+    // This syntax is unusual, but very helpful here.  It's equivalent to `unnamedObject[NOTIFICATION_DROP_LEDGER_FIREFOX] =`
+    [NOTIFICATION_DROP_LEDGER_FIREFOX]: {
+      ...UI_NOTIFICATIONS[NOTIFICATION_DROP_LEDGER_FIREFOX],
+      title: t('notificationsDropLedgerFirefoxTitle'),
```

Comment:
Looks like this announcement was brought in accidentally (the associated localized messages don't exist)

Reply:
Amended the commit and force pushed, should be the only commit that changed.